optimization/optimization_pr.md

```diff
@@ -0,0 +1,399 @@
+</a>
+<p>  </p>
+</td>
+
+</tr>
+
+</table>
+Parameter values and relative changes in the optimization shown in Figure 10.
+</div>
+
+## Optimizing with less smoothing
+
+Even though our optimization has dropped the simulation's RMSE compared to the suprathreshold data to 10.40, we are still using a large smoothing window (the default for perceptual-level threshold simulations) of 30 ms. The suprathreshold data has sharper features that we may want to optimize to, notably the second small peak between the first proximal and distal inputs. To explore what the simulation output looks like with a smaller smoothing window, click "Set Parameters" -> "Run", select the "Analysis" tab and then enter 15 for "Dipole Smooth Window (ms)". Change the simulation name to indicate that smoothing has changed (e.g. append "window_15ms"), and then click "Run Simulation" to see the output. As expected, RMSE rises, but some other features in the simulated dipole not matching the experimental data become apparent. In this case, the negative trough shifts significantly to the left.
+
+Let's attempt to fit parameters using the less-smoothed output by running another optimization round. Leave the optimization configuration at the default number of simulations and parameter ranges. After the optimization completes, you should see a plot like Figure 12 below, where the RMSE is reduced to 8.49, a very good fit.
+
+<div style="background-color:rgba(0, 0, 0, 0); margin-top:20px; text-align: center; vertical-align: middle; margin-bottom:40px;">
+
+<h3>Figure 12</h3>
+
+<a href="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image18.png">
+<img src="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image18.png" alt="image18" width=80%/>
+</a>
+<p>Fit after optimization using simulation trials with a 15 ms smoothing window
+</p>
+</div>
+
+Note that there were minimal changes to the first and second proximal inputs. For those inputs changes were 75% or less, but for the distal evoked input, AMPA weights changed significantly. The plot reflects this in that the regions around the proximal inputs changed minimally, but the negative trough immediately following the distal input is fit very closely by the optimized parameters. The downward deflection follows the slope of the experimental data closely and nearly reaches its minimum at the same time, around 75 ms.
+
+Through a total of 4 optimizations from a baseline parameter set designed for a different experimental scenario, we have found parameters matching new experimental data.
+
+<div style="background-color:rgba(0, 0, 0, 0); text-align:center; vertical-align: middle; margin-top:20px; margin-bottom:40px">
+
+<table style="border-collapse: collapse; border: none;">
+
+<tr style="border: none;">
+<h3>Figure 13</h3>
```

Comment:
Looks much better with the adjusted smoothing window. In future, HNN might allow people to optimizing the smoothing window itself...